### PR TITLE
[WIP] PCT-1480

### PR DESCRIPTION
**Pull request by mpaolino**
Modifications to cloud-protocol for instance management in PCTv3, WIP.

### CODE REVIEW

File: proto/service.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ type UniqueInstance struct {
 
 type ServiceInstance struct {
 	Service    string // internal (agent, log, data) or and ExternalService
-	InstanceId uint   // external service instance ID: server-5, mysql-9, etc.
+	InstanceId string // external service instance UUID.
```
Comment (cfsalguero) [inline proto/service.go]: Why not InstanceUUID?

Comment (mpaolino) [inline proto/service.go]: Good question, I kept the Id for naming consistency with spec:

```go
{
Type:     "OS"
Prefix:   "os"
Name:     "bm-cloud-db01"
ParentId: ""
Id:       "w39d23A9"
Properties: {
             distro:  "Ubuntu"
             version: “12.04”
             arch:    “64-bit”
            }
}
```

Comment (mpaolino) [inline proto/service.go]: I saw the contradiction, specs says one thing, your code another. But in any case, I don't think this should even exist in PCTv3 (?) . The change was primarily to be able to compile the thing and run instance tests until they pass.

**Comment by mpaolino** [comment]
Closing, as we are using only instance trees for sync now, a new PR will be created.

**Pull request closed without merging**